Tools/autotest/quadplane.py

```diff
@@ -1335,6 +1335,117 @@ def VTOLQuicktune(self):
         self.remove_installed_script(applet_script)
         self.reboot_sitl()
 
+    def PrecisionLanding(self):
+        '''VTOL precision landing'''
+        applet_script = "plane_precland.lua"
+
+        self.install_applet_script(applet_script)
+
+        target = mavutil.location(-27.27440414, 151.28984285, 342.8, 0)
```

Comment:
```suggestion
        here = self.mav.location()
        target = self.offset_location_ne(here, metres_north=5, metres_east=2)
```
... or whatever offsets make sense.

This will reduce the change of a cascading failure if we land (slightly) elsewhere

Reply:
changed